forms-flow-web/src/components/ServiceFlow/list/ServiceTaskListView.js

```diff
@@ -272,7 +272,7 @@ const ServiceTaskListView = React.memo((props) => {
                    
 
                       {vissibleAttributes?.taskVisibleAttributes?.priority &&
-                        <Col xs={2} className="d-flex">
+                        <Col xs={1} className="d-flex">
```

Comment:
@Ajay-aot can you please check using the task variable dropdown (filter with task variables) . I think that some UI breaking will occur.
also check in case of "priority" replacing task variable with assignee .

Reply:
in the screen shot you can see the arrow button come down

Reply:
@Ajay-aot

Reply:
@Ajay-aot  We were displaying six variables along with a dropdown button on the first row, Here there are just five.

Reply:
ok